### ISSUE TITLE
Please don't print anything to the console when the plugin is loaded

### ISSUE THREAD

**Issue by rwols** [20Tauri/DoxyDoxygen#144]
It's a bit annoying that this message is printed:

```
------==== DoxyDoxygen plugins loaded ====------
0 sub-modules to reload
```
It seems to serve no purpose because we can already see this:
```
reloading python 3.3 plugin DoxyDoxygen.Doxy
```

**Comment by 20Tauri** [comment]
It's not exactly the same logs. Those logs were added to resolve the tricky problem encountered on package update. On package update, Sublime Text / Package Control mixed the new and the old versions ... This caused unstability and crash (that was most no single file plugins ask to restart sublime text on upgrade)
To solve this ZipImporter has been modified

Now, its less useful (no crash has been reported for a very very long time, and no anomaly caused by this).

So, those logs may be removed in a next release.

**Comment by rwols** [comment]
> So, those logs may be removed in a next release.

Cool. Personally I just always restart ST when a package gets upgraded. I have been conditioned to do this over time. In general, only log something to console once something fails and never when something succeeds ;)

**Comment by 20Tauri** [comment]
Fixed in the last evolution version (0.76.4)
